<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lecteurvue</class>
 <widget class="QMainWindow" name="lecteurvue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lecteurvue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>861</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="btnLancerDiapo">
       <property name="text">
        <string>Lancer Diaporama</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titreDiapo">
       <property name="text">
        <string>Titre du diaporama</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnArreterDiapo">
       <property name="text">
        <string>Arrêter le diaporama</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>861</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="image">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titreImage">
       <property name="text">
        <string>Titre de l'image</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="catImage">
       <property name="text">
        <string>Catégorie</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>309</y>
      <width>861</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnPrec">
       <property name="text">
        <string>Précédent</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerEntrePrecEtSuiv">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSuiv">
       <property name="text">
        <string>Suivant</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuParam_tres">
     <property name="title">
      <string>Paramètres</string>
     </property>
     <addaction name="actionCharger_Diaporama"/>
     <addaction name="actionChanger_Vitesse_Defilement"/>
    </widget>
    <addaction name="menuParam_tres"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos_de"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="actionCharger_Diaporama">
   <property name="text">
    <string>Charger Diaporama</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionA_propos_de">
   <property name="text">
    <string>A propos de</string>
   </property>
  </action>
  <action name="actionChanger_Vitesse_Defilement">
   <property name="text">
    <string>Changer la vitesse de défilement</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
